api/src/main/java/io/opentelemetry/metrics/DoubleGauge.java

```diff
@@ -52,6 +52,18 @@
  */
 @ThreadSafe
 public interface DoubleGauge extends Gauge<BoundDoubleGauge> {
+
+  /**
+   * Sets the given value.
+   *
+   * <p>The value added is associated with the current {@code Context}.
```

Comment:
and the provided labelset? probably everywhere?

Reply:
yes, that's a good idea. I'll add that to the javadoc on all of these

Reply:
done